dump/mapping/bufr_adpupa_prepbufr.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/env python3
+import os
+import sys
+sys.path.append('/scratch1/NCEPDEV/da/Praveen.Singh/HERA/bufr-query/build/lib')
```

Comment:
Get rid of these sys.path.append lines.

Reply:
@nicholasesposito thanks, removed these lines.